docs/source/api/express-middleware.mdx

```diff
@@ -17,7 +17,7 @@ The `expressMiddleware` function expects you to set up HTTP body parsing and COR
 
 > See [Configuring CORS](../security/cors#specifying-origins) for guidance on configuring the CORS behavior of your project.
 
-The `expressMiddleware` function accepts two arguments. The first **required** argument is an instance of `ApolloServer` that has been `start`ed:
+The `expressMiddleware` function accepts two arguments. The first **required** argument is an instance of `ApolloServer` that has called the `start` method:
```

Comment:
`ApolloServer` doesn't call it, so this reads a bit off too. Here's my suggestion:
```suggestion
The `expressMiddleware` function accepts two arguments. The first **required** argument is an instance of `ApolloServer` that has been started by calling its `start` method:
```

Reply:
Ok, awesome makes sense. Should I keep the change to the comments?